docs/core/upgrade.md

```diff
@@ -0,0 +1,98 @@
+<!--
+order: 15
+-->
+
+# In-Place Store Migrations
+
+Upgrade your app modules smoothly with custom in-place migration logic. {synopsis}
+
+The Cosmos SDK currently has two ways to perform upgrades. The first way is by exporting the entire application state to a JSON file using the `export` CLI command, making changes, and then starting a new binary with the changed JSON file as the genesis file. The second way is by performing upgrades in place, significantly decreasing the time needed to perform upgrades for chains with a larger state. The following guide will provide you with the necessary information in order to setup your application to take advantage of in-place upgrades.
+
+## Genesis State
+
+Each app module's consensus version must be saved to state on the application's genesis. This can be done by adding the following line to the `InitChainer` method in `app.go`
```

Comment:
```suggestion
When starting a new chain, each module's consensus version must be saved to state on the application's genesis. This can be done by adding the following line to the `InitChainer` method in `app.go`
```